core-bundle/src/Resources/contao/languages/pl/default.xlf

```diff
@@ -1533,7 +1533,7 @@ Ten e-mail został wygenerowane przez Contao. Nie odpowiadaj na niego.
       </trans-unit>
       <trans-unit id="MSC.save">
         <source>Save</source>
-        <target>Zapisz</target>
+        <target>Zachowaj</target>
```

Comment:
Please revert back to `Zapisz` as it seems more common in various software in general. At least to me 😉 

Could you then please change `Zachowaj` to `Zapisz` in other places as well (e.g. `Zachowaj i nowy`)?

Reply:
`Save` is fine to me to be `Zapisz`, I will correct all of its occurences.